examples/src/main/java/com/mapbox/navigation/examples/core/BaseRouterActivityJava.java

```diff
@@ -141,7 +141,7 @@ public void onNewLocationClick() {
   }
 
   private void newOrigin() {
-    if (mapboxMap != null) {
+    if (mapboxMap != null && mapboxMap.getStyle() != null) {
```

Comment:
may be better `show`/`hide` fab button when style is ready?

Reply:
do we have the same fix for `BaseRouterActivityKt`?

Reply:
done